integration-cli/docker_cli_run_test.go

```diff
@@ -4849,3 +4850,34 @@ func (s *DockerSuite) TestRunEmptyEnv(c *check.C) {
 	c.Assert(err, checker.NotNil)
 	c.Assert(out, checker.Contains, expectedOutput)
 }
+
+// #28658
+func (s *DockerSuite) TestSlowStdinClosing(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
Can we enable this test for Windows as well, although it is not related to the fix for container_unix.go?